apis/core/bootstrap/shapes/table.ts

```diff
@@ -41,6 +41,12 @@ ${shape('table/create')} {
       ${sh.order} 40 ;
       ${dash.editor} ${editor.ColorPicker} ;
     ] ;
+    ${sh.property} [
+      ${sh.name} "Columns to map" ;
+      ${sh.path} ${csvw.column} ;
+      ${sh.order} 50 ;
+      ${dash.hidden} ${true} ;
```

Comment:
I quickly tried and realized it would take quite some time to figure out, so unless you already know how I could change the values in the select boxes when the source is changed, I'll just concentrate on other things first and come back to it later.